docs/rule/kotlin_library.soy

```diff
@@ -0,0 +1,130 @@
+{namespace kotlin_library}
+
+/***/
+{template .soyweb}
+  {call buck.page}
+    {param title: 'kotlin_library()' /}
+    {param navid: 'rule_kotlin_library' /}
+    {param prettify: true /}
+    {param description}
+      A rule that is used to define a set of Kotlin files that can be compiled
+      together.
+    {/param}
+    {param content}
+
+{call buck.rule}
+{param status: 'UNFROZEN' /}
+{param overview}
+A <code>kotlin_library()</code> rule is used to define a set of Kotlin
+files that can be compiled together. The main output of a
+{sp}<code>kotlin_library()</code> rule is a single JAR file containing all
+of the compiled class files and resources.
+{/param}
+
+{param args}
+
+{call buck.arg}
+  {param name: 'name' /}
+  {param desc}
+  The name of the rule.
+  {/param}
+{/call}
+
+{call buck.arg}
+  {param name: 'srcs' /}
+  {param default: '[]' /}
+  {param desc}
+  The set of <code>.kt</code> or <code>.kts</code> files to compile for this rule.
```

Comment:
My preference would be that doc change be part of the mixed-source change.

In-memory Kotlin compiler support has been a long time coming. I don't want the publication of the documentation to be gated or delayed by completion and testing on another feature.